### ISSUE TITLE
Leigh - How-To Image Carousel

### ISSUE THREAD

**Issue by qjhart** [UCDavisLibrary/fin-example-repository#26]
review how-to document and image carousel to determine garden path for digital collections.  Determine if the image carousel will work in place of the image list.

**Comment by qjhart** [comment]
@jrmerz / @ldragoon Let me know what you think about the review above.  My initial inclination is to not worry about updating the `ImageList` that we currently use but that this could offer a nice description for a Garden Path. 

We could mock up a simple rendering for these, or go straight to @wrenaria for input on that.  We could also extend the 'ListItem`s to include ucdlib specific controls if needed.

**Comment by jrmerz** [comment]
Will this path ever have splits/forks?  Otherwise it seems like a nice fit.

**Comment by qjhart** [comment]
Right, it's tied to being linear now.  I have a few ideas to make  it more a 1`) tree or 2) a graph, but neither would be schema.org entities in that case.

**Comment by qjhart** [comment]
@EricNebeker this issue points to the spec, and gives an example of the code.  I think we can make the description part markdown as well.
